### PR TITLE
Add ephemeral support for group chats

### PR DESCRIPTION
**Pull request by hakonbo**
#### APK
[Download build #7819](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7819/artifact/build/artifact/wire-dev-PR280-7819.apk)
[Download build #7829](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7829/artifact/build/artifact/wire-dev-PR280-7829.apk)
[Download build #7830](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7830/artifact/build/artifact/wire-dev-PR280-7830.apk)
[Download build #7834](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7834/artifact/build/artifact/wire-dev-PR280-7834.apk)

### CODE REVIEW

**Comment by bjoernherbig** [comment]
build again

File: wire-ui/src/main/java/com/waz/zclient/ui/cursor/CursorLayout.java [hunk 1]
```diff
@@ -137,7 +137,9 @@ public void updated(IConversation conversation) {
             }
 
             if (!isEditingMessage &&
-                conversation.getType() == IConversation.Type.ONE_TO_ONE &&
+                (conversation.getType() == IConversation.Type.ONE_TO_ONE ||
+                 conversation.getType() == IConversation.Type.GROUP) &&
+                conversation.isActive() &&
```
Comment (deanrobertcook) [inline wire-ui/src/main/java/com/waz/zclient/ui/cursor/CursorLayout.java]: I think you can just drop the check altogether, no?

**Comment by vova-lohika** [comment]
Is this PR with assets v3?
Could not receive ephemeral images (1:1/group) - they are empty.

**Comment by deanrobertcook** [comment]
build again

File: wire-ui/src/main/java/com/waz/zclient/ui/cursor/CursorLayout.java (hunk 1 above)
Comment (hakonbo) [inline wire-ui/src/main/java/com/waz/zclient/ui/cursor/CursorLayout.java]: I guess the ```isActive``` part is needed, but perhaps not the type check... I couldn't be bothered checking if we used the cursor layout for invites etc ;) I'll do it now and remove it not needed...

**Comment by hakonbo** [comment]
@vova-lohika please file asset v3 issues separately...

**Comment by hakonbo** [comment]
build again

**Comment by vova-lohika** [comment]
ephemeral msg sent to group is deleted for everyone (all users in group) once 1st user read the message on device with assets v2 / old build.

**Comment by mykola-mokhnach** [comment]
Ephemeral message times for the most recent conversation message is expected to be visible by default for both outgoing and incoming message, but it is visible for incoming messages only for now.

**Comment by deanrobertcook** [comment]
Created ticket for last issue and will handle in next release:
https://wearezeta.atlassian.net/browse/AN-4652

**Pull request merged**